core/src/main/java/me/lokka30/treasury/plugin/core/command/subcommand/DownloadLatestSubcommand.java

```diff
@@ -0,0 +1,43 @@
+package me.lokka30.treasury.plugin.core.command.subcommand;
+
+import me.lokka30.treasury.plugin.core.command.CommandSource;
+import me.lokka30.treasury.plugin.core.command.Subcommand;
+import me.lokka30.treasury.plugin.core.config.messaging.Message;
+import me.lokka30.treasury.plugin.core.config.messaging.MessageKey;
+import me.lokka30.treasury.plugin.core.config.messaging.MessagePlaceholder;
+import me.lokka30.treasury.plugin.core.utils.QuickTimer;
+import me.lokka30.treasury.plugin.core.utils.Utils;
+import me.lokka30.treasury.plugin.core.utils.downloader.PluginDownloader;
+import org.jetbrains.annotations.NotNull;
+
+public class DownloadLatestSubcommand implements Subcommand {
```

Comment:
should add a javadoc here with `@author MrIvanPlays`

Reply:
done